ucc/tests/test_compile.py

```diff
@@ -130,6 +108,59 @@ def run(self, dag):
     post_compiler_circuit = compile(cirq_circuit, custom_passes=[HtoX()])
     assert_same_circuits(post_compiler_circuit, CirqCircuit(X(qubit)))
 
+    def test_compile_target_device_opset():
```

Comment:
@jordandsullivan -- I stumbled on this but realized the indenting of this and the next test function make them inner functions inside the `test_custom_pass` function and they are never run. If I restore the formatting properly, the `test_compile_target_device_coupling_map` function is failing on the current version.

There is a potentially another issue with target device still. I'm confirming and will open a consolidated issue for both, but flagging for the moment here.